Source/WebCore/loader/SubresourceIntegrity.cpp

```diff
@@ -161,6 +162,87 @@ static Vector<EncodedResourceCryptographicDigest> strongestMetadataFromSet(Vecto
     return result;
 }
 
+static Ref<FormData> createReportFormData(const String& type, const URL& url, const String& userAgent, const Function<void(JSON::Object&)>& populateBody)
+{
+    auto body = JSON::Object::create();
+    populateBody(body);
+
+    // https://www.w3.org/TR/reporting-1/#queue-report, step 2.3.1.
+    auto reportObject = JSON::Object::create();
+    reportObject->setObject("body"_s, WTFMove(body));
+    reportObject->setString("type"_s, type);
+    reportObject->setString("user_agent"_s, userAgent);
+    reportObject->setInteger("age"_s, 0); // We currently do not delay sending the reports.
```

Comment:
This comment is mysterious and needn't be. Comments need to say why the code does what it does, not just what it does. Why don't we delay? Why does the specification allow for delay? Do we need to change this in the future? Need to touch on this to make it a good comment, but still good to be terse if we can.

Reply:
Changed it to indicate reasoning with a pointer to the relevant spec.